src/Features/CSharp/Portable/EmbeddedLanguages/CSharpEmbeddedLanguageCodeFixProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.EmbeddedLanguages.LanguageServices;
+using Microsoft.CodeAnalysis.EmbeddedLanguages;
+
+namespace Microsoft.CodeAnalysis.CSharp.EmbeddedLanguages
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(CSharpEmbeddedLanguageCodeFixProvider)), Shared]
+    internal class CSharpEmbeddedLanguageCodeFixProvider : AbstractEmbeddedLanguageCodeFixProvider
+    {
+        public CSharpEmbeddedLanguageCodeFixProvider()
+            : base(CSharpEmbeddedLanguageProvider.Instance)
```

Comment:
All interesting code exists in base type.   This type exists only for exporting and to pass along the right EmbeddedLanguageProvider to the base type.